src/app/bisoncoin/src/components/Button.vue

```diff
@@ -24,7 +29,8 @@ export default class Button extends Vue {
   transition: 0.2s linear;
   margin-top: 100px;
   margin-bottom: 50px;
-  border: 1px solid black;
+  border: 1px solid $border-color;
```

Comment:
we might be able to add the "1px solid" to $border-color in the style file, so then $border-color includes all variables needed

Reply:
Fixed!